### ISSUE TITLE
Configuration uses `spacemacs|add-company-hook`, which is removed in spacemacs/develop

### ISSUE THREAD

**Issue by RichardDRJ** [Ruin0x11/lsp-intellij-spacemacs#1]
I've got an alternative config for users of the develop branch of Spacemacs, in [this branch](https://github.com/CatharticMonkey/lsp-intellij-spacemacs/tree/develop), but I can't see an easy way to raise a PR which creates a branch at the same time - I think this change should be on a `develop` branch in this repo, so that both bleeding-edge and non-bleeding-edge Spacemacs users have an easy option.

This change also introduces Spacemacs-like keybindings for lsp-intellij.

Thanks,
Richard

**Comment by Ruin0x11** [comment]
I wonder how other Spacemacs packages would deal with this. It would be nice if we had a version that worked on both versions of Spacemacs. If not then I can make the current repo the stable branch and merge the bleeding-edge changes into master, then add somehing in the README.

**Comment by beyondpie** [comment]
@CatharticMonkey  @Ruin0x11 
I see both of you have big differences in spacemacs/lsp-intellij.  Is it good to combine them?  And send a PR to spacemacs?

**Comment by RichardDRJ** [comment]
@Ruin0x11 Yeah, I like that. Built-in layers deal with it by being bundled with the correct version, but I have no idea how 3P layers usually would. I've added a version check to differentiate between `master`'s HEAD and `develop`'s HEAD, and load the correct config based on that, which is in [this PR](https://github.com/Ruin0x11/lsp-intellij-spacemacs/pull/2).

@songpeng Yep, the PR I've raised on this repo combines the config. I'm not sure it's worth sending a PR to Spacemacs right now, though; there seems to be some churn coming up to the 0.300 release, and then once that's done there's talk of changing how contributions are reviewed and merged, to remove the bottleneck of a single maintainer. I'd be inclined to wait a little bit before trying to get this merged into mainline Spacemacs.
